{{cookiecutter.project_slug}}/dev.yml

```diff
@@ -19,7 +19,8 @@ services:
       dockerfile: ./compose/django/Dockerfile-dev
     command: /start-dev.sh
     depends_on:
-      - postgres
+      - postgres{% if cookiecutter.use_mailhog == 'y' %}
```

Comment:
Does it mean one may not worry about line breaks in jinja2 should `trim_blocks` and `lstrip_blocks` settings be set?